corehq/ex-submodules/dimagi/utils/csv.py

```diff
@@ -49,27 +49,15 @@ def __iter__(self):
 
 class UnicodeWriter:
     """
-    A CSV writer which will write rows to CSV file "f",
-    which is encoded in the given encoding.
+    A CSV writer which will write rows to CSV file "f" using utf-8.
     """
-    def __init__(self, f, dialect=csv.excel, encoding="utf-8", **kwds):
+    def __init__(self, f, dialect=csv.excel, **kwds):
         # Redirect output to a queue
         self.queue = io.BytesIO()
```

Comment:
nit: this is not used anymore. could be removed

Reply:
https://github.com/dimagi/commcare-hq/pull/19581